recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "5.5" %}
+{% set version = "6.1.1" %}
 
 package:
   name: coverage
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/c/coverage/coverage-{{ version }}.tar.gz
-  sha256: ebe78fe9a0e874362175b02371bdfbee64d8edc42a044253ddf4ee7d3c15212c
+  sha256: b8e4f15b672c9156c1154249a9c5746e86ac9ae9edc3799ee3afebc323d9d9e0
```

Comment:
sha256 for v6.2 is `e2cad8093172b7d1595b4ad66f24270808658e11acf43a8f95b41276162eb5b8`

Reply:
The `sha256` number was updated:

https://github.com/AnacondaRecipes/coverage-feedstock/blob/update_to_6.1.1/recipe/meta.yaml#L9